### PR TITLE
Rerender to start building against Numpy 1.13

### PR DESCRIPTION
**Pull request by pkgw**

### CODE REVIEW

**Comment by ocefpaf** [comment]
You may want to try https://conda-forge.org/docs/meta.html#building-against-numpy

You can even use np18 (or np19, not sure which is the minimum version for `casacore`) for all the python versions.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -24,13 +24,17 @@ requirements:
     - gcc  # [osx]
     - boost 1.63.*  # note this is intentionally NOT the latest version; synced with casacore which has special needs
     - casacore 2.2.*
-    - numpy x.x
+    - numpy 1.7.*  # [py27]
+    - numpy 1.9.*  # [py35]
+    - numpy 1.11.*  # [py36]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: You will need to rerender again to update the build matrix. BTW, you can probably use:

`numpy 1.8.*` and `numpy >=1.8` for build and run respectively.

Comment (pkgw) [inline recipe/meta.yaml]: Right, thanks. Third time's the charm? We'll see ...

Comment (ocefpaf) [inline recipe/meta.yaml]: The trick here is to use the oldest possible `numpy` to build. I did not find `casacore` lower limit in the repo but `1.8` is usually safe.

Also, after I wrote the docs I submitted `numpy 1.8` and `numpy 1.9` updates to all Pythons, so we are not limited to `numpy 1.9.*  # [py35]` and `numpy 1.11.*  # [py36]`. The only problem is that we don't have Windows packages for those yet, so `py36`+`win ` is still limited to `numpy 1.11.*  # [py36]`.

Comment (pkgw) [inline recipe/meta.yaml]: OK. I think I get the gist but from your description I'm still a bit confused about what requirements I could put instead, so I'll stick with the document ones for now.

Comment (ocefpaf) [inline recipe/meta.yaml]: > I'm still a bit confused about what requirements I could put instead

Sorry for that. 

> so I'll stick with the document ones for now.

I am updating the docs now though :smile: 

When I wrote that we did not have numpy 1.8 and 1.9 available for all pythons, that is why we ended up with a `numpy` for each python. The new docs will be:

```yaml
build:
    - numpy 1.8.*  # [not (win and py36)]
    - numpy 1.11.*  # [win and py36]
run:
    - numpy >=1.8  # [not (win and py36)]
    - numpy >=1.11  # [win and py36]
```

And, if you project allows, you can go as low as `numpy 1.7` for `Python 2.7`, but for simplicity I guess that 1.8 is good enough.

**Pull request merged**